toolchain/lexer/tokenized_buffer_test.cpp

```diff
@@ -1183,10 +1171,11 @@ TEST_F(LexerTest, PrintToken) {
   llvm::raw_string_ostream print_stream(print_output);
   buffer.Print(print_stream);
   llvm::StringRef print = print_stream.str();
+  EXPECT_THAT(GetAndDropLine(print), StrEq("["));
   EXPECT_THAT(GetAndDropLine(print),
-              StrEq("token: { index: 0, kind: 'IntegerLiteral', line: 1, "
+              StrEq("{ index: 0, kind: 'IntegerLiteral', line: 1, "
                     "column: 1, indent: 1, spelling: '0x9', value: `9`, "
-                    "has_trailing_space: true }"));
+                    "has_trailing_space: true },"));
```

Comment:
Would it make sense to add a closing bracket check too?
```suggestion
                    "has_trailing_space: true },"));
  EXPECT_THAT(GetAndDropLine(print), StrEq("]"));
```

Reply:
It's a possibly misleading test! It doesn't check for FileEnd right now, so it can't check for ].

I was tempted to change it, but that's going to drag me down a rathole of rewriting these tests to lit to make them easier to maintain in general, and I worry if I do that too much chandlerc will amp up the pressure on me to build a better testing approach.

Reply:
(easier to maintain because, I'd literally be running the code and copying the output to get all the FileEnd details correct, which is exactly what lit_autoupdate does)